### ISSUE TITLE
Bug in stations.csv comment parsing: Doesn't take CSV module parsing behavior into account

### ISSUE THREAD

**Issue by EvilRenegade** [coderholic/pyradio#28]
Take the first line of the stock stations.csv. Append the text `", "` to it, like so:
```python
# Find lots more stations at http://www.iheard.com ", "
```
**Expected result:** It doesn't matter, because it's in a comment.
**Actual result:** Your entire playlist just vanished.

The reason for this is that **the comment skipping code relies on the CSV module parsing the line first**, and the CSV parser will react to legitimate CSV syntax elements.
In this particular example, it will parse the first column from the beginning of the line to the comma, assume the quotation mark before the comma to be the end of column 1, and the quotation mark after the comma to be the start of column 2. It will then continue to parse this assumed "column 2" until it finds closing quotation marks or reaches EOF.

The result is that the entire playlist is in row[1] of the very first line of the file, which is skipped because row[0] of that line is then identified as a comment.

According to the CSV module documentation
> csvfile can be any object which supports the iterator protocol and returns a string each time its __next__() method is called — file objects and list objects are both suitable.

I suggest a pre-parsing stage where you read stations.csv as a normal text file first, kick out all comments and empty lines (#27), and then feed the remaining pure list of station entries into the CSV module, thus making sure that the comments don't interfere with the parsing.

**Comment by coderholic** [comment]
Heh, nice little bug :) 

I think you're the first person to go digging so deep into the stations file and the relating parsing code. I'm happy to leave this one as is though. It seems unlikely that someone would accidentally enter the exact 4 character string required to break this without knowing what they're doing. Therefore the extra effort of pre-parsing the stations file doesn't feel worth it. Nice find though!

**Comment by EvilRenegade** [comment]
I understand your reasoning, but I would like to point out that I did, in fact, enter that string by accident.

I was creating a stations.csv containing all the Digitally Imported stations (including their sister sites), and left a comment for my future self with the code I used.
This code included the concatenation 
```javascript
$foo[$i].name + ", " + $foo[$i].playlist
```
to generate the individual CSV rows.

Suddenly, no stations. :O
